pyaedt/generic/plot.py

```diff
@@ -1042,12 +1044,29 @@ def plot(self, export_image_path=None):
         else:
             self.pv.isometric_view()
         self.pv.camera.zoom(self.zoom)
+        if export_image_path:
+            path_image = os.path.dirname(export_image_path)
+            format = export_image_path.split(".")[-1]
+            root_name = os.path.splitext(os.path.basename(export_image_path))[0]
+        else:
+            path_image = tempfile.gettempdir()
+            format = "png"
+            root_name = "Image"
+
+        def s_callback():
+            """save screenshots"""
+            exp = os.path.join(path_image,
+                               "{}{}.{}".format(root_name, datetime.now().strftime("%Y_%M_%d_%H-%M-%S"), format))
```

Comment:
```suggestion
                               root_name + datetime.now().strftime("%Y_%M_%d_%H-%M-%S") + format)
```
if you use `os.path.splitext`